rollup.config.js

```diff
@@ -3,31 +3,87 @@ import { terser } from "rollup-plugin-terser";
 
 export default [
   {
-    input: "src/single-spa-react.js",
+    input: `src/single-spa-react.js`,
+    output: [
+      {
+        file: `lib/umd/single-spa-react.js`,
+        format: "umd",
+        name: "singleSpaReact",
+        sourcemap: true,
+      },
+      {
+        file: `lib/system/single-spa-react.js`,
+        format: "system",
+        sourcemap: true,
+      },
+      {
+        file: `lib/esm/single-spa-react.js`,
+        format: "esm",
+        sourcemap: true,
+      },
+    ],
+    plugins: [babel(), terser()],
+  },
+  {
+    input: "./src/single-spa-react.js",
     output: {
-      file: "lib/single-spa-react.js",
-      format: "umd",
-      name: "singleSpaReact",
+      file: `./lib/es2015/single-spa-react.js`,
+      format: "esm",
       sourcemap: true,
     },
+    plugins: [
+      babel({
+        envName: "esm",
```

Comment:
We do not have an `esm` env defined in [our babelrc](https://github.com/single-spa/single-spa-react/blob/master/.babelrc). I am happy to be shown otherwise, but as far as I know this configuration is not doing anything.

https://babeljs.io/docs/en/options#envname

https://babeljs.io/docs/en/options#env

Reply:
Ok